options.go

```diff
@@ -48,7 +48,7 @@ type Options struct {
 	Truncate            bool
 	Logger              Logger
 	Compression         options.CompressionType
-	EventLogging        bool
+	Logging             bool
```

Comment:
This is also not necessary. The user can set the logger via the `opt.Logger` option.